package.json

```diff
@@ -18,6 +18,7 @@
     "react-bootstrap": "^2.9.1",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.49.0",
+    "react-loader-spinner": "^6.1.6",
```

Comment:
No need for a third-party package for loader spinner, it adds up to the JavaScript bundle for no good benefit.
You can choose a spinner from this website, and it will provide to the CSS for it which you can easily customize to your need:
[CSS Loaders](https://cssloaders.github.io/)